### DISCUSSION TITLE
What's the difference between 2 "inpaint"

### DISCUSSION THREAD

**Discussion by sololll** [Mikubill/sd-webui-controlnet#1447] [General]
I'm kind of confused, there are 2 "inpaint" .
One is the A1111's own inpaint in i2i tab, the other is in Controlnet.
Can someone tell me :
What's the difference ？
Which one is better or better in some way ？
Can they be used together and how ?

I'm trying .
If experienced people can share their experience, that would be much appreciated.

**Comment by lllyasviel** [comment]
inpaint_only does not change unmasked area in txt2img

**Reply by sololll** [reply]
> inpaint_only does not change unmasked area in txt2img

Wow, got a reply so quickly, thank you!
"inpaint_only does not change unmasked area in txt2img"   I just tried a few times, it's pretty good, thank you again!
